internal/cmd/egctl/config_cmd.go

```diff
@@ -35,6 +36,10 @@ func newConfigCommand() *cobra.Command {
 	return cfgCommand
 }
 
+func ratelimitCommand() *cobra.Command {
```

Comment:
why not use `ratelimitConfigCommand` directly?

Reply:
I have thought about this problem, but I hope to write it like the `func proxyCommand()` function for backward compatibility.